code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -205,86 +205,86 @@
-			/obj/item/weapon/gun/smg/standard_machinepistol = 250,
-			/obj/item/ammo_magazine/smg/standard_machinepistol = 1000,
-			/obj/item/weapon/gun/smg/ppsh = 250,
-			/obj/item/ammo_magazine/smg/ppsh = 1000,
-			/obj/item/ammo_magazine/smg/ppsh/extended = 500,
+			/obj/item/weapon/gun/smg/standard_smg = -1,
+			/obj/item/ammo_magazine/smg/standard_smg = -1,
+			/obj/item/weapon/gun/smg/standard_machinepistol = -1,
+			/obj/item/ammo_magazine/smg/standard_machinepistol = -1,
+			/obj/item/weapon/gun/smg/ppsh = -1,
+			/obj/item/ammo_magazine/smg/ppsh = -1,
+			/obj/item/ammo_magazine/smg/ppsh/extended = -1,
 		),
 		"Sidearm" = list(
-			/obj/item/weapon/gun/pistol/standard_pistol = 250,
-			/obj/item/ammo_magazine/pistol/standard_pistol = 1000,
-			/obj/item/weapon/gun/pistol/standard_heavypistol = 250,
-			/obj/item/ammo_magazine/pistol/standard_heavypistol = 1000,
-			/obj/item/weapon/gun/revolver/standard_revolver = 250,
-			/obj/item/ammo_magazine/revolver/standard_revolver = 1000,
-			/obj/item/weapon/gun/pistol/standard_pocketpistol = 1000,
-			/obj/item/ammo_magazine/pistol/standard_pocketpistol = 1000,
-			/obj/item/weapon/gun/pistol/vp70 = 250,
-			/obj/item/ammo_magazine/pistol/vp70 = 1000,
-			/obj/item/weapon/gun/pistol/plasma_pistol = 250,
-			/obj/item/ammo_magazine/pistol/plasma_pistol = 1000,
+			/obj/item/weapon/gun/pistol/standard_pistol = -1,
+			/obj/item/ammo_magazine/pistol/standard_pistol = -1,
+			/obj/item/weapon/gun/pistol/standard_heavypistol = -1,
+			/obj/item/ammo_magazine/pistol/standard_heavypistol = -1,
+			/obj/item/weapon/gun/revolver/standard_revolver = -1,
+			/obj/item/ammo_magazine/revolver/standard_revolver = -1,
+			/obj/item/weapon/gun/pistol/standard_pocketpistol = -1,
+			/obj/item/ammo_magazine/pistol/standard_pocketpistol = -1,
+			/obj/item/weapon/gun/pistol/vp70 = -1,
+			/obj/item/ammo_magazine/pistol/vp70 = -1,
+			/obj/item/weapon/gun/pistol/plasma_pistol = -1,
+			/obj/item/ammo_magazine/pistol/plasma_pistol = -1,
 		),
 		"Specialized" = list(
 			/obj/item/weapon/gun/flamer/marinestandard = 4,
 			/obj/item/ammo_magazine/flamer_tank/backtank = 4,
 			/obj/item/ammo_magazine/flamer_tank/large = 20,
 			/obj/item/ammo_magazine/flamer_tank = 20,
 			/obj/item/weapon/shield/riot/marine = 6,
-			/obj/item/weapon/powerfist = 100,
-			/obj/item/weapon/throwing_knife = 500,
+			/obj/item/weapon/powerfist = -1,
+			/obj/item/weapon/throwing_knife = -1,
 			/obj/item/ammo_magazine/standard_smartmachinegun = 4,
 		),
 		"Grenades" = list(
-			/obj/item/explosive/grenade/frag = 600,
+			/obj/item/explosive/grenade/frag = -1,
 			/obj/item/explosive/grenade/frag/m15 = 50,
 			/obj/item/explosive/grenade/incendiary = 50,
 			/obj/item/explosive/grenade/cloakbomb = 50,
 			/obj/item/explosive/grenade/drainbomb = 10,
-			/obj/item/explosive/grenade/mirage = 100,
-			/obj/item/storage/box/m94 = 200,
+			/obj/item/explosive/grenade/mirage = -1,
+			/obj/item/storage/box/m94 = -1,
```

Comment:
probably a balance discussion to be made here, do we want unlimited m15, incendiary and clockbomb?

unlimited drainbomb is probably terrible though